packages/compiler/core/options.ts

```diff
@@ -6,6 +6,10 @@ export interface CompilerOptions {
   noEmit?: boolean;
   watchForChanges?: boolean;
   serviceCodePath?: string;
+  /**
+   * when true , turn off the build check.
```

Comment:
So far, I can't figure out from this comment nor the prior internal PR why this is needed. Can you provide more info? My gut feeling is that it's weird to have this option on the CLI, but maybe I'm missing something.

Reply:
This option if for skipping the lint checks  which will be ran in the `onBuild` of emitter,
I am totally agreed to use the --diagnosticLevel to specify which diagnostics should be reported, as now all the lint rules are warnings .